### PR TITLE
add: alert component

### PR DESCRIPTION
**Pull request by vishalg8454**
Added the first component into the component library.

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.alert{
+    border-radius: 0.5rem;
```
Comment (viru2001) [inline components/alert.css]: for smaller units you can use px

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    padding: 0.5rem 1rem;
+    margin: 0.8rem;
+    font-size: 1.1rem;
+}
+.alert-primary{
+    background-color: rgba(23, 97, 180, 0.397);
+    color: rgba(6, 60, 122, 0.904);
```
Comment (viru2001) [inline components/alert.css]: you can declare variables for colors

File: docs.css [hunk 3]
```diff
@@ -1,38 +1,63 @@
...
-    gap: 0.5rem;
-    align-items: center;
-}
-.sidebar-list-item{
-    color: var(--color-primary);
-}
\ No newline at end of file
+@import url("style.css");
+:root {
+  --color-font: #333;
```
Comment (viru2001) [inline docs.css:3]: you can give proper name here . for e.g. `--black-color`

File: docs.css [hunk 4]
```diff
@@ -1,38 +1,63 @@
...
+}
+.doc-container {
+  display: flex;
+}
+main {
+  padding: 1rem;
+}
```
Comment (viru2001) [inline docs.css:10]: you should avoid giving styles to html elements directly. you can use class or id for it

File: docs.html [hunk 5]
```diff
@@ -40,27 +40,23 @@
...
-                doloremque iure nesciunt recusandae quidem sequi. Consequuntur corporis unde quis id quaerat non,
-                laudantium itaque harum voluptatum?
+                <h2 class="component-title">Alert</h2>
+                <p class="component-description">Alert is used to show important information to the user without
+                    interrupting the user's flow. The five types of alerts available are 'alert-primary',
+                    'alert-secondary', 'alert-danger', 'alert-success' and 'alert-failure'</p>
+                <div class="component-demo">
+                    <div class="alert alert-primary">This is an example of primary alert.</div>
+                    <div class="alert alert-secondary">This is an example of secondary alert.</div>
+                    <div class="alert alert-success">✅ This is an example of success alert.</div>
```
Comment (viru2001) [inline docs.html]: you can use font awesome icons here

Comment (vishalg8454) [reply docs.html]: done.

**Review by Developers-codz — APPROVED** [review]
Looks good to me!

**Pull request merged**